### DISCUSSION TITLE
Best practice to handle content width on large screens ?

### DISCUSSION THREAD

**Discussion by finkarder** [quasarframework/quasar#15849] [General - Components / Directives / etc]
## Problem

When displaying Quasar-built websites on large screens, the default is for containers to expand and fill the whole width.
When screens are excessively large the result can easily be disappointing.

## Workaround

A common practice is to limit displays with `max-width`, so I am doing this in `app.sass`:

```
#q-app, header
   max-width: 1280px
   margin: 0 auto

body
  background: #000
```

## Question:

Is there an alternative best practice for Quasar ?

Thanks.

**Comment by suleimanas** [comment]
there is no one solution for doing it. I've seen many ways some developers make a container component other do exactly like yours. for me I like to have a container class:
```
.container
   max-width: 1280px
   margin: 0 auto
```
